dtu-hpc-python3/tensorflow.patch

```diff
@@ -21,11 +22,27 @@ index e7808ca..55a6464 100644
    // TODO(jeff,sanjay): For open source version, replace this with something
    // that logs through the env or something and fill in appropriate time info.
    fprintf(stderr, "%c %s:%d] %s\n", "IWEF"[severity_], fname_, line_,
+diff --git a/tensorflow/workspace.bzl b/tensorflow/workspace.bzl
+index 06e16cd..5484138 100644
+--- a/tensorflow/workspace.bzl
++++ b/tensorflow/workspace.bzl
+@@ -228,7 +228,10 @@ def tf_workspace(path_prefix = "", tf_repo_name = ""):
+
+   native.new_http_archive(
+     name = "zlib_archive",
+-    url = "http://zlib.net/zlib-1.2.8.tar.gz",
++    urls = [
```

Comment:
I changed it back but with the new url ([commit](http://zlib.net/zlib-1.2.8.tar.gz)). The old url was dead, because they moved the file from 
```http://zlib.net/zlib-1.2.8.tar.gz``` 
to
```http://zlib.net/fossils/zlib-1.2.8.tar.gz```